src/App.tsx

```diff
@@ -1,24 +1,116 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useMemo } from 'react';
+import debounce from 'lodash.debounce';
+import { TodoCreate } from './components/TodoCreate';
+import { TodoInfo } from './components/TodoInfo';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { client } from './utils/fetchClient';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
 
-const USER_ID = 0;
+const USER_ID = '6757';
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>();
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const countNotComplited = useMemo(() => todosFromServer?.some(
+    (todo: Todo) => todo.completed === false,
+  ), [todosFromServer]);
+
+  const countComplited = useMemo(() => todosFromServer?.some(
+    (todo: Todo) => todo.completed === true,
+  ), [todosFromServer]);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const fetchTodos = (url: string) => {
+    client
+      .get(url)
+      .then((todos) => {
+        setTodosFromServer(todos as Todo[]);
+      })
+      .catch(() => setErrorMessage('Unable to update a todo'));
+  };
+
+  const askTodos = debounce((url) => fetchTodos(url), 1000);
```

Comment:
do not understand the reason to simulate a debounce here)

Reply:
I ask server in many place my code. If dont use, page rerender more time, without sence. More comfortable use debonce then.